addons/istio/1.5.x/istio-1.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: istio
+  labels:
+    kubeaddons.mesosphere.io/name: istio
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.4-1"
+    appversion.kubeaddons.mesosphere.io/istio: "1.5.4"
+    appversion.kubeaddons.mesosphere.io/kiali: "1.15.1"
+    appversion.kubeaddons.mesosphere.io/jaeger: "1.16.0"
+    stage.kubeaddons.mesosphere.io/kiali: Preview
+    stage.kubeaddons.mesosphere.io/jaeger: Preview
+    endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
+    endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
+    docs.kubeaddons.mesosphere.io/istio: "https://istio.io/docs/"
+    docs.kubeaddons.mesosphere.io/kiali: "https://istio.io/docs/tasks/telemetry/kiali/"
+    docs.kubeaddons.mesosphere.io/jaeger: "https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/"
+spec:
+  namespace: istio-system
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: cert-manager
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  cloudProvider:
+    - name: aws
+      enabled: false
+    - name: azure
+      enabled: false
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: false
+  kudoReference:
+    package: istio-operator
+    repo: https://kudo-private.storage.googleapis.com
+    version: 1.5.4
```

Comment:
You might want to cut a format release at https://github.com/mesosphere/kudo-istio, and re-post the artifact to gcloud with the operator version as <app version>-<operator version> (`1.5.4-0.1.0` in this case).

This would allow us to do revisioning, in case there are more params (e.g., sds, mtls) added in the future against the same app version 1.5.4

Reply:
It is that way just corrected the operator version. Thanks for pointing that out